plugins/mcreator-core/procedures/entity_add_modifier.json

```diff
@@ -0,0 +1,70 @@
+{
+  "args0": [
+    {
+      "type": "field_resourcelocation",
+      "name": "name",
+      "text": "name"
+    },
+    {
+      "type": "field_data_list_selector",
+      "name": "attribute",
+      "datalist": "attributes",
+      "customEntryProviders": "attribute"
+    },
+    {
+      "type": "input_value",
+      "name": "value",
+      "check": "Number"
+    },
+    {
+      "type": "field_dropdown",
+      "name": "operation",
+      "options": [
+        [
+          "Addition",
+          "ADD_VALUE"
+        ],
+        [
+          "Multiply base",
+          "ADD_MULTIPLIED_BASE"
+        ],
+        [
+          "Multiply total",
+          "ADD_MULTIPLIED_TOTAL"
+        ]
+      ]
+    },
+    {
+      "type": "input_value",
+      "name": "entity",
+      "check": "Entity"
+    },
+    {
+      "type": "field_checkbox",
+      "name": "permanent",
+      "checked": false

```

Comment:
I'd set this field checked by default. Users probably want the modifiers to persist after world reloads in most cases.